Dockerfile

```diff
@@ -18,11 +18,10 @@ COPY internal/ internal/
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
Why the change of the base image?

Reply:
Other kubevirt projects use `ubi8` as a base for containers.

Reply:
Ah, very well, nice catch